locator/topology.hh

```diff
@@ -52,9 +50,8 @@ public:
 
     /**
      * Returns true iff contains given endpoint.
-     * Excludes pending endpoints if `with_pending == pending::no`.
      */
-    bool has_endpoint(inet_address, pending with_pending) const;
+    bool has_endpoint(inet_address) const;
```

Comment:
Also this.

Reply:
Other than update_normal_tokens, that didn't need it, all other callers used `pending::yes`